gradle/validation/dependency-check.gradle

```diff
@@ -0,0 +1,14 @@
+// This adds OWASP vulnerability validation of project dependencies
+// Not part of 'check' task by default, must be called explicitly, e.g. gradlew dependencyCheckAnalyze
+// Start build with -Pvalidation.owasp.fail=true to fail build on owasp errors (CVSS >= 7)
+// Start build with -Pvalidation.owasp.skip=true to skip OWASP checks during 'check' phase
+
+dependencyCheck {
```

Comment:
Does it automatically hook to all projects or is it configured for root project only?

Reply:
I don’t know. Have not tested other than root.

Reply:
You can try to run gradlew -p lucene/core ... and see what happens. :)
If it's just applicable to the root project (which it may be) then I'd wrap it in a configure(rootProject) { ... } closure so that it's explicit in the code.

Reply:
> Task 'dependencyCheckAnalyze' not found in project ':lucene:core'.

Will wrap it